<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PageExtioWgtClass</class>
 <widget class="QWidget" name="PageExtioWgtClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>403</width>
    <height>538</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="ExtioSettingsBox">
     <property name="title">
      <string>ExtIO setup</string>
     </property>
     <property name="checkable">
      <bool>false</bool>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QCheckBox" name="checkBox">
        <property name="text">
         <string>IO pin is output</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QCheckBox" name="checkBox_2">
        <property name="text">
         <string>Invert</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>ExtIO mode</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QComboBox" name="comboBox">
        <item>
         <property name="text">
          <string>Do nothing</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Stop on input</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Power off on input</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Movr on input</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Home on input</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Alarm on input</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="comboBox_2">
        <item>
         <property name="text">
          <string>Out always off</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Out always on</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Out active when moving</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Out active in Alarm</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Out active when motor is on</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Out active when motor is found</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>checkBox</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>checkBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>comboBox</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>65</x>
     <y>50</y>
    </hint>
    <hint type="destinationlabel">
     <x>140</x>
     <y>147</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>comboBox_2</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>262</x>
     <y>50</y>
    </hint>
    <hint type="destinationlabel">
     <x>268</x>
     <y>177</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
